### DISCUSSION TITLE
completed task report

### DISCUSSION THREAD

**Discussion by esquires** [nvim-neorg/neorg#242] [Q&A] [answered]
I often would like to see a report of what has been done over the last `n` days. What do you think about adding a "#done.date" to the task tags so we can generate that type of report?

**Comment by danymat** [comment, marked as answer]
Hello, sorry for missing this post, and I just stumbled upon it.

I think that it can provide a great addition, and will be able to easily do logging/archiving system that's planned for 0.2 in GTD.
I'm better off using `#time.done ...` which is more consistent with other time related tags.

**Reply by esquires** [reply]
sounds good to me! I'll create a feature request and deferring to 0.2 works on my end. Happy to assist as well.
